Test/client_test.py

```diff
@@ -66,6 +66,20 @@ def test_get_build_logs_no_ssl(self):
             print(result)
         self.assertTrue("error" not in result)
 
+    def test_get_build_logs_customCert(self):
+        with open("./test.pem",'w'):
+            client = Algorithmia.client(api_key=os.environ.get('ALGORITHMIA_API_KEY'), ca_cert="./test.pem")
```

Comment:
I think it should pickup api_key from environment variable automatically, no?

Also what exactly is this testing? The certificate is empty?